unity/Assets/Plugins/Countly/CountlyManager.cs

```diff
@@ -278,20 +285,23 @@ protected void Suspend()
     }
 
 #region Utility Methods
-    protected void ProcessConnectionQueue()
-    {
-      if ((_isProcessingConnection == true) ||
-          (ConnectionQueue.Count <= 0))
-      {
-        return;
-      }
 
-      _isProcessingConnection = true;
-      StartCoroutine(_ProcessConnectionQueue());
-    }
+	protected void ProcessConnectionQueue()
+	{
+		if ((_isProcessingConnection == true) ||
+		    (ConnectionQueue.Count <= 0))
+		{
+			return;
+		}
+			
+		_isProcessingConnection = true;
+		
+		StartCoroutine(_ProcessConnectionQueue());
+	}
 
     protected IEnumerator _ProcessConnectionQueue()
     {
+	  int retry = 0;
```

Comment:
this looks to me for another PR. and it would be good to have this option exported so that you can select 1, 2, 3, 4, 5, ...